src/react-dnd-ax/drag-n-drop-container.js

```diff
@@ -216,57 +239,47 @@ const DragNDropContainer = (WrappedComponent) => {
         isDragging: false,
         isKeyboardMoving: true,
         sourceIndex: index,
-        keyInsertIndex: index + 1,
+        keyInsertIndex: index,
         curPreview: preview,
       })
     }
 
     onKeyChangeOrder = (e) => {
-      const {items} = this.props
+      const {items} = this.props;
```

Comment:
Please remove the added semicolons, as we were following code standards `https://github.com/standard/standard`.

Reply:
Fixed